tests/testDipoleFitter.py

```diff
@@ -74,34 +83,35 @@ def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
             xcenNeg=self.xc - self.offsets,
             ycenNeg=self.yc - self.offsets,
             flux=self.flux, fluxNeg=self.flux,
+            noise=2.,  # Note the input noise - this affects the relative tolerances used.
             gradientParams=self.gradientParams)
 
         self.catalog = self.testImage.detectDipoleSources()
+        self.rtol = 0.01  # This is okay given the default noise of 2. set above.
 
 
 # First, test the algorithm itself (fitDipole()):
```

Comment:
Again: top level comments aren't helpful. Either this goes into the class docstring, or just get rid of it. What is "first" here anyway?